cmd/soroban-cli/src/commands/ledger/latest.rs

```diff
@@ -0,0 +1,54 @@
+use crate::{commands::global, config::network, rpc};
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    Serde(#[from] serde_json::Error),
+    #[error(transparent)]
+    Network(#[from] network::Error),
+    #[error(transparent)]
+    Rpc(#[from] rpc::Error),
+}
+
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, clap::ValueEnum, Default)]
+pub enum OutputFormat {
+    /// Text output of network info
+    #[default]
+    Text,
+    /// JSON result of the RPC request
+    Json,
+    /// Formatted (multiline) JSON output of the RPC request
+    JsonFormatted,
+}
+
+#[derive(Debug, clap::Parser)]
+pub struct Cmd {
+    pub seq: Option<i64>,
```

Comment:
This option looks unused, is that right?

Reply:
good catch!